examples/language/gpt/train_gpt_pp_demo.py

```diff
@@ -51,6 +51,10 @@ def get_data(batch_size, seq_len, vocab_size):
     return input_ids, attention_mask
 
 
+def get_tflops(batch_size, seq_len, step_time, model_numel):
```

Comment:
use common utils for both pp and gemini flops calc?

Reply:
Done